### PR TITLE
Awesome books: refactor to use class in Java Script

### PR DESCRIPTION
**Pull request by migo1**
### MILESTONES

- Create and use javascript classes
- Create class methods to add, remove and display books

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,25 +4,56 @@
...
+    <form id="bookForm" class="form container mt-4">
+      <div class="form-group">
+        <input
+          class="form-control form-control-lg"
+          type="text"
+          name="title"
+          id="title"
+          required
+          placeholder="Title"
+        />
```
Comment (codepantha) [inline index.html:45]: - [ ] Please reduce the width of the form container in order to be consistent with the layout of the wireframe
- [ ] Please add a horizontal line before the form just like we have in the wireframe design attached below 👇 
### Wireframe
image
### Your Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Refactored app to use JavaScript classes 📌 
✔️ Add and remove functionality still working 📌 
✔️ Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (migo1) [reply index.html:45]: @codepantha noted

**Review by ShadyShawkat — APPROVED** [review]
## Hi @migo1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**